commands.js

```diff
@@ -1111,7 +1111,8 @@ var commands = exports.commands = {
 	promotehelp: ["/promote [username], [group] - Promotes the user to the specified group. Requires: & ~"],
 
 	globaldemote: 'demote',
-	demote: function () {
+	demote: function (target) {
```

Comment:
@Rhythms88, I appreciate your desire to help, but I think you should try harder not to say wrong things. In particular, when "correcting" other people, you really need to be careful. It might help to phrase your sentences to be less sure of yourself, compare:

> This isn't really a fix, just a change to allow for parsing `/help demote`

with

> Are you sure this is a fix? I might be wrong, but I think it's just a change to allow for parsing `/help demote`.

This way, it's less embarrassing for you and frustrating for other people when you're wrong.